### ISSUE TITLE
6.J. If Board overturns Committee recommendation

### ISSUE THREAD

**Issue by yvonneng** [amiaopensource/amia-elections#13]
Suggest more compelling language than "Committee reserves the right to express its dissenting opinion to the AMIA membership" if Board overturns Committee recommendation. Should be that Committee "must" inform the membership if the board does not accept the Committee's recommendation.

**Comment by dericed** [comment]
I don't think so. The Board may reject the Committee's recommendation for a reason that the Committee agrees with: typos, biased presentation of candidate names, member status. I don't think the Committee needs to inform the membership if the Board is rejecting the nomination due to some types of mistakes.

**Comment by yvonneng** [comment]
But @dericed , this section is about the results of the election, not the nominations, right? To me, "overturn" sounds more severe than just sending something back for typo corrections -- it would be the Board overturning the recommendations about the results of the election, which seems like it would have serious consequences.

**Comment by dericed** [comment]
My apologies, I was thinking about a different point in communication. Reading again. The "recommendation" in this context to the Committee recommending "whether or not the election should be certified".
So either:
- The Committee advises that the Election IS NOT certified (not at quorum, policy violations, etc) and the Board certifies it anyway.
- The Committee advises that the Election IS certified and the Board has a reason to not.

Personally, I think the first scenario has a more compelling reason for a MUST, whereas the second one may just follow with a new Committee recommendation. I agree that 'overturn' should be more severe than mistakes (like the EC miscalculating quorum or typos) and if this is proposed then the language should be clearly about how the Board ultimately acts upon the recommendation and not necessarily about the aspects of the communication of the recommendation.

Also with requirements language, I'd like to see a normative reference such as RFC2119 so there's some clarify about mandates.

I'm interested in hearing other thoughts on this.

**Comment by bpgraney** [comment]
I don’t understand the basis for these certification and dissent sections.

With the EC acting here as the Board-appointed inspectors of the election (as per [CCC 5615](http://leginfo.legislature.ca.gov/faces/codes_displaySection.xhtml?lawCode=CORP&sectionNum=5615.)), the EC itself either certifies or invalidates the election based on its oversight of the process.  It informs the Board and the membership of this determination.  Its report isn’t a recommendation for the Board to approve or reject, it’s a finding: “Any report or certificate made by the inspectors of election is prima facie evidence of the facts stated therein.”

I can see recommending that the EC as a courtesy share its report with the Board before releasing it to the membership, but without any call for official action by the Board to certify or overturn the result.  

The Board has no special power to overturn an election as suggested in the manual.  See also CCC [5527](http://leginfo.legislature.ca.gov/faces/codes_displaySection.xhtml?lawCode=CORP&sectionNum=5527.) and [5617](http://leginfo.legislature.ca.gov/faces/codes_displaySection.xhtml?lawCode=CORP&sectionNum=5617.) regarding actions against an election result.
